DataScience/Analytics/beta/check_performance.py

```diff
@@ -181,7 +194,9 @@ def _check_performance(data, verbose=True):
         print('Correct Match not in the Set:', len(incorrect))
 
     results.append(len(correct_in_set))
+    results.append(new_uprns)
```

Comment:
Both are ints; line 171 shows that new_uprns = len() which returns an int. 

In general, this is not a great way of doing this, but it makes the plotting later easier. One could probably find a cleaner way of doing what's needed here (i.e. to capture results).